velox/type/TimestampConversion.cpp

```diff
@@ -709,12 +735,29 @@ int64_t fromTimeString(const char* str, size_t len) {
   int64_t microsSinceMidnight;
   size_t pos;
 
-  if (!tryParseTimeString(str, len, pos, microsSinceMidnight, true)) {
+  if (!tryParseTimeString(
+          str,
+          len,
+          pos,
+          microsSinceMidnight,
+          TimestampParseMode::kPrestoCast)) {
```

Comment:
why don't we forward `parseMode` here?

Reply:
fromTimeString is a test-only utility that doesn't take (and doesn't need) parseMode. I have a TODO to move it to test-only code to avoid confusion.